portal/src/pages/my/privacy.vue

```diff
@@ -146,19 +150,54 @@ export default {
     },
     submitButtonLabel() {
       if (
+        !this.$store.getters.hasGivenCommunityPermission &&
```

Comment:
Dit kun je ook in dit `mapGetters` hierboven zetten. Dan kun je die `$store.getters` weglaten.

Reply:
👍